lua/gruvbox_crisp/init.lua

```
@@ -0,0 +1,75 @@
+  vim.g.terminal_color_14 = t.bright_cyan    or p.hint
+  vim.g.terminal_color_15 = t.bright_white   or p.fg0
+end
+
+function M.setup(opts)
+  opts = opts or {}
+  M.options = vim.tbl_deep_extend("force", vim.deepcopy(defaults), opts)
+end
+
+function M.load()
+  local o = M.options
+  vim.o.termguicolors = true
```

Luacheck (1.2.0)

[warning] mutating non-standard global variable 'vim'

(W112)